.env.example

```diff
@@ -1 +1,6 @@
-DATABASE=
+DB_USERNAME_DEVELOPMENT=

```

Comment:
Please remove all DB_... and add DATABASE_URL and DATABASE_TEST_URL as that's what we will be using.